corehq/ex-submodules/couchforms/models.py

```diff
@@ -248,6 +247,7 @@ def found_in_multiselect_node(self, xpath, option):
 
     @memoized
     def get_sync_token(self):
+        from casexml.apps.phone.models import SyncLog
```

Comment:
yeah there was an import cycle that only manifested itself when running management commands  in the couchforms app